SWIPs/swip-chunk-and-hasher.md

```diff
@@ -0,0 +1,129 @@
+---
+SWIP: <To be assigned>
+title: Swarm node implementer spec - Chunk and hasher
+author: Louis Holbrook <dev@holbrook.no> (https://holbrook.no)
+status: Draft
+type: Track Specs
+created: 2019-09-14
+---
+
+## Abstract
+
+This SWIP is a part of a general specification of a Swarm node. The SWIP system is used for convenience only.
+
+## Motivation
+
+Enable other implementations of the swarm node.
+
+## Backwards Compatibility
+
+N/A
+
+## Test Cases
+
+N/A
+
+## Implementation
+
+N/A
+
+## Copyright
+
+Copyright and related rights waived via CC0
+
+## Specification
+
+Data stored in Swarm consists of \texttt{Chunks}.
+
+\subsection{Chunk Size}
+
+The size of a chunk is the digest size of the \texttt{Base Hash} multiplied by an arbitary exponential of 2. Currently the size is defined as 4096 bytes \footnote{$2^7 * 32$}
```

Comment:
if this is just a constant and can be defined as such with a standard definition field

this is needlessly hard to parse imo, e.g. compare with something like https://github.com/ethereum/eth2.0-specs/blob/dev/specs/simple-serialize.md#constants which is easier to scan without verbiage

Reply:
if we want to have motivations or rational for picking certain constants perhaps they can be part of separate section at bottom, similar to https://github.com/ethereum/eth2.0-specs/blob/dev/specs/networking/p2p-interface.md#design-decision-rationale